lib/src/model.dart

```diff
@@ -3843,21 +3581,24 @@ class Package extends Nameable implements Documentable {
     }
 
     List<String> messageParts = [warningMessage];
-    if (warnable != null)
-      messageParts.add(
-          "${warnablePrefix} ${warnableStrings.item1}: ${warnableStrings.item2}");
+    if (warnable != null) {
+      messageParts
+          .add("${warnablePrefix} ${warnableStrings.name}: ${warnableStrings
+              .location}");
+    }
     if (referredFrom != null) {
       for (Locatable referral in referredFrom) {
         if (referral != warnable) {
-          Tuple2<String, String> referredFromStrings =
-              nameAndLocation(referral);
+          var referredFromStrings =
```

Comment:
I'll defer to @jcollins-g here, but removing the type here makes the code strictly less readable. You get no info from the right abut what the type is, and removing it from the left leaves you w/ nothing to go on. In this case it's not just something simple like a `String`, but a more complicated generic type - `Tuple2<String, String>`. (similarly with the `final` assignment above)

Reply:
in both cases, I'm explicitly initializing `new _LocatableStrings`. Not sure how this is less readable.

🤷‍♂️

Reply:
Sorry, I read this as replacing `Tuple2<String, String>` with `var`, not `_LocatableStrings` with var.